web/src/components/CurrentSong.js

```diff
@@ -47,15 +54,32 @@ function CurrentSong({song, dj, startTime, offset}) {
           <span className="dj-name">{dj.name}</span>
         </div>
         <div className="skip">
-          <button className="btn" disabled>
+          <button className="btn" onClick={doVoteSkip}>
             Skip
           </button>
-          <span>{/* TODO */}</span>
+          <span className={skipTextClassName}>
+            {skips > 0 ? `${skips}/${votesNeeded} votes` : `0/${votesNeeded} votes`}

```

Comment:
bumping: `skips` should default 0

Reply:
this means:
1. you don't have to special case this line for 0 skips
1. you will need to update the reducer

Reply:
🐇